0008-managed-k8s-via-cluster-api/0008-managed-k8s-via-cluster-api.md

```diff
@@ -0,0 +1,7 @@
+# Managed K8s in our KaaS product
+
+As a User I would like to provision EKS or AKS clusters via Giant Swarm.
+
+Benefits: We use the building block we get from the cloud providers and save effort on our side. ClusterAPI allows this as well.
```

Comment:
Another benefit that we would get here with CAPI is that theoretically cluster adoption, i.e. taking over existing AKS/EKS clusters into our management, also should work.